charts/crossplane-efs/templates/xrd.yaml

```diff
@@ -0,0 +1,76 @@
+# Crossplane Composite Resource Definition (XRD) describing
+# the type and schema of the corresponding Composite Resource.
+# See https://negz.github.io/crossplane.github.io/docs/v1.4/reference/composition.html#compositeresourcedefinitions for details
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: elasticfilesystems.{{ .Values.xrd.apiGroup }}
+spec:
+  group: {{ .Values.xrd.apiGroup }}
+  names:
+    kind: ElasticFilesystem
+    plural: elasticfilesystems
+  claimNames:
+    kind: ElasticFilesystemClaim
+    plural: elasticfilesystemclaims
+  versions:
+    - name: {{ .Values.xrd.version }}
+      served: true
+      referenceable: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                id:
+                  type: string
+                  description: "Unique id used for tagging all dependent components."
+                parameters:
+                  type: object
+                  properties:
+                    region:
+                      type: string
+                      description: "AWS Region in which to create network resources."
+                      enum: ["eu-west-1", "eu-central-1", "us-east-1"]
+                    performanceMode:
```

Comment:
throughputMode and provisionedThroughputInMibps are still missing in the xrd and composition. 
can you adapt that from the current setup in the integration environment?

Reply:
Puh, I copied from master. Need to find the changes

Reply:
see infra deployment repo

Reply:
done